client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -59,12 +59,47 @@ export interface Props {
 	currentDomain?: DomainSuggestion;
 }
 
-const DomainPicker: FunctionComponent< Props > = ( {
-	onDomainSelect,
-	onDomainPurchase,
-	onClose,
-	currentDomain,
-} ) => {
+const FreeDomainIcon = () => (
+	<Icon
+		icon={ () => (
+			<svg
+				width="15"
+				height="15"
+				viewBox="0 0 15 15"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<path
+					d="M13.2878 8.60833L8.61291 13.2844C8.4918 13.4057 8.34798 13.5019 8.18968 13.5676C8.03137 13.6332 7.86169 13.667 7.69032 13.667C7.51895 13.667 7.34926 13.6332 7.19096 13.5676C7.03265 13.5019 6.88884 13.4057 6.76773 13.2844L1.16699 7.68876V1.16699H7.68706L13.2878 6.76919C13.5307 7.01358 13.667 7.34417 13.667 7.68876C13.667 8.03335 13.5307 8.36395 13.2878 8.60833V8.60833Z"
+					stroke="#008A20"
+					strokeWidth="1.5"
+					strokeLinecap="round"
+					strokeLinejoin="round"
+				/>
+				<circle cx="4.50033" cy="4.50033" r="0.833333" fill="#008A20" />
+			</svg>
+		) }
+	/>
+);
```

Comment:
Could maybe use the [tag icon](https://github.com/WordPress/gutenberg/blob/a97f31c4a80f4d945d10cfd6a7a5301baad9dabf/packages/icons/src/library/tag.js) from the `@wordpress/icons` npm

Reply:
Oh, Calypso isn't using the latest (G2 compliant) version yet :confused: -- it would look like this with icons from `@wordpress/icons`:

![image](https://user-images.githubusercontent.com/96308/78550557-913b3400-7804-11ea-9d99-7ac3537dd191.png)

This is even with `@wordpress/icons` v1.2.0 installed in `client/` -- unfortunately, v1.1.0 is still installed at root `node_modules/` level, and it seems like Gutenboarding is picking up the latter :confused: 

For reference, this is what it should look like:

![image](https://user-images.githubusercontent.com/96308/78554910-4a513c80-780c-11ea-9b24-89dbbbf764f4.png)

So we'll probably have to wait for #39644 to land before we can use `@wordpress/icons` v1.2.0 :confused:

Reply:
Filed #40803 to track this.